src/metadata/ColumnMetadata.ts

```diff
@@ -815,8 +815,10 @@ export class ColumnMetadata {
         } else {
             // no embeds - no problems. Simply return column name by property name of the entity
             if (this.relationMetadata && this.referencedColumn) {
-                const relatedEntity =
-                    this.relationMetadata.getEntityValue(entity)
+                const relatedEntity = this.relationMetadata.getEntityValue(
+                    entity,
+                    this.relationMetadata.isLazy,
```

Comment:
Internally `getEntityValue` method is checking `isLazy` too, but to avoid regressions I prefer to change only this piece of code.

The only other method that is using that flag is [this one](https://github.com/typeorm/typeorm/blob/fe71a0c3e441bc5fc68eec78c575d7ab5cc81bad/src/entity-manager/EntityManager.ts#L298)